SyncUp.Data/Models/Group.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.IdentityModel.Tokens;
+
+namespace IntelliTect.SyncUp.Data.Models;
+
+public class Group : TenantedBase
+{
+    public long GroupId { get; init; }
+
+    [MaxLength(500)]
+    public required string Name { get; set; }
```

Comment:
I have a preference to also add [Required] on top of these, but let's not block the PR for it.